src/java/com/palantir/cassandra/utils/CountingCellIterator.java

```diff
@@ -103,6 +106,9 @@ private void count(Cell cell) {
         } else if (ExpiringCell.class.isAssignableFrom(cell.getClass())) {
             droppableTtls++;
         } else if (cell.getLocalDeletionTime() < gcBeforeSeconds) {
+            if (metrics != null) {
+                metrics.droppableTombstones.mark();
```

Comment:
can you explain why this is correct to mark the metrics here and likewise in `SliceQueryFilter#collectReducedColumns`? previously, we set these at the level of `ColumnFamilyStore ` in `getColumnFamily`. are there other codepaths that use the filter that aren't from `getColumnFamily`?

Reply:
discussed offline -- this is strictly an improvement as long as we're not double-counting. if do double-count someday, there is no risk of badness